Parser/pegen.c

```diff
@@ -400,16 +397,17 @@ _PyPegen_raise_error_known_location(Parser *p, PyObject *errtype,
 
     if (!error_line) {
         Py_ssize_t size = p->tok->inp - p->tok->buf;
-        if (size && p->tok->buf[size-1] == '\n') {
-            size--;
-        }
         error_line = PyUnicode_DecodeUTF8(p->tok->buf, size, "replace");
         if (!error_line) {
             goto error;
         }
     }
 
-    Py_ssize_t col_number = byte_offset_to_character_offset(error_line, col_offset);
+    Py_ssize_t col_number = col_offset;
```

Comment:
``Py_BuildValue`` might be using this variable uninitialized

Reply:
Could you elaborate what you mean? `col_number` is always initialized to `col_offset`, which is passed to the function so there is no code path where `col_number` is uninitialized.

Reply:
Please ignore my comment. I misread the line.